From a4f923c7b5b04c30d334a887d706ea1d4595121d Mon Sep 17 00:00:00 2001
From: David Wu <david.wu@rock-chips.com>
Date: Wed, 23 Dec 2020 16:52:44 +0800
Subject: [PATCH] arm64: dts: rockchip: rk3566: Fix the rgmii delayline for
 evb3 and evb5

After changing the drive strength, fix the current
delayline.

Change-Id: I2aca4ddef692214a9c2e2cf7bae261cea04027b1
Signed-off-by: David Wu <david.wu@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3566-evb3-ddr3-v10.dtsi | 2 +-
 arch/arm64/boot/dts/rockchip/rk3566-evb5-lp4x-v10.dtsi | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3566-evb3-ddr3-v10.dtsi b/arch/arm64/boot/dts/rockchip/rk3566-evb3-ddr3-v10.dtsi
index cbb83b5754e9..e71f3acd7637 100644
--- a/arch/arm64/boot/dts/rockchip/rk3566-evb3-ddr3-v10.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3566-evb3-ddr3-v10.dtsi
@@ -93,7 +93,7 @@
 		     &gmac1m0_rgmii_clk_level2
 		     &gmac1m0_rgmii_bus_level3>;
 
-	tx_delay = <0x59>;
+	tx_delay = <0x41>;
 	rx_delay = <0x2e>;
 
 	phy-handle = <&rgmii_phy1>;
diff --git a/arch/arm64/boot/dts/rockchip/rk3566-evb5-lp4x-v10.dtsi b/arch/arm64/boot/dts/rockchip/rk3566-evb5-lp4x-v10.dtsi
index f2a67c92ca52..590990c4c5e1 100644
--- a/arch/arm64/boot/dts/rockchip/rk3566-evb5-lp4x-v10.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3566-evb5-lp4x-v10.dtsi
@@ -149,7 +149,7 @@
 		     &gmac1m0_rgmii_clk_level2
 		     &gmac1m0_rgmii_bus_level3>;
 
-	tx_delay = <0x59>;
+	tx_delay = <0x41>;
 	rx_delay = <0x2e>;
 
 	phy-handle = <&rgmii_phy1>;
-- 
2.35.3

